product-features/farming/how-to.md

```diff
@@ -0,0 +1,208 @@
+
+{% hint style="info" %}
+To be completely sure, you can check:
+
+* Transaction is requested by **"Alex app" (app.alexlab.co)**
+* The amounts you will transfer to the smart contract, covered by [Stacks post conditions](https://docs.stacks.co/stacks-101/post-conditions).&#x20;
+{% endhint %}
+
+<figure><img src="../../.gitbook/assets/farming/stake-tokens-function-args.png" alt="" width="375"><figcaption><p>Wallet pop-up with function arguments and confirmation button.</p></figcaption></figure>
+
+### Step 6
+
+Wait for the transaction to be confirmed on the network.
+
+<figure><img src="../../.gitbook/assets/farming/tx-broadcasted.png" alt="" width="375"><figcaption></figcaption></figure>
+
+{% hint style="info" %}
+Recommended to track transaction status:
+
+* Turn on [Telegram notifications](https://t.me/stacks\_tx\_notification\_bot), you will get notified when the transaction is confirmed.
+* Search for the transaction on the [ALEX Explorer](https://app.alexlab.co/explorer).
+* Check your address activity on the wallet.
+{% endhint %}
+
+<!-- Real tx: https://explorer.hiro.so/txid/0x0445e0ce8a1872e0aea18971e421fcc137048f91473ddbdf931c0660d5ed087c?chain=mainnet -->
+
+<div>
+
+<figure><img src="../../.gitbook/assets/farming/tg-tx-pending.png" alt="" width="352"><figcaption><p>Telegram message with transaction pending status.</p></figcaption></figure>
+
+ 
+
+<figure><img src="../../.gitbook/assets/farming/tg-tx-success.png" alt="" width="358"><figcaption><p>Telegram message with transaction success status.</p></figcaption></figure>
+
+</div>
+
+<div>
+
+<figure><img src="../../.gitbook/assets/farming/leather-tx-pending.png" alt="" width="375"><figcaption><p>Transaction pending displayed on Leather wallet.</p></figcaption></figure>
+
+ 
+
+<figure><img src="../../.gitbook/assets/farming/leather-tx-success.png" alt="" width="375"><figcaption><p>Transaction completed.</p></figcaption></figure>
+
+</div>
+
+### Step 7
+
+After successfully staking your LP tokens in a farm, you will be able to see your active farms in the "My Farms" panel on the main [Farms page](https://app.alexlab.co/farm).
+
+<figure><img src="../../.gitbook/assets/farming/my-farms.png" alt=""><figcaption><p>Example of the "My Farms" panel. Here you will find all your active farms; click on any of them for detailed information.</p></figcaption></figure>
+
+By clicking on a farm, you will access the "My Farming" dashboard for that specific farm, which includes detailed metrics. On the right side of the dashborad, you will see that the current cycle has no earnings and no farming tokens. This is due to the cooldown period, farming will start in the next upcoming cycle. This is why it's convenient to stake for long periods: every time you stake, you must wait for the current cycle to end before you start generating rewards in the next cycle.
```

Comment:
Original sentence [This is due to the cooldown period, farming will start in the next upcoming cycle.]
Suggestion:
The reason you can't join the current reward cycle is that it had already started prior to your participation. However, once you successfully stake your LP into the farm, it gets registered for the next cycle. This assures you a proportional share of the farm rewards based on the number of LP tokens you have staked.

Reply:
Corrected